nilearn/decomposition/dict_learning.py

```diff
@@ -0,0 +1,187 @@
+"""
+DictLearning
```

Comment:
Alternative: "Perform Dictionary Learning: divide brain maps by linear combinations of basis functions with sparse coefficients"